src/libclipper/src/logging.cpp

```diff
@@ -0,0 +1,44 @@
+#include <spdlog/spdlog.h>
+#include <spdlog/sinks/ostream_sink.h>
+#include <boost/algorithm/string.hpp>
+
+#include <clipper/logging.hpp>
+
+namespace clipper {
+
+  Logger::Logger() {
+    spdlog::set_async_mode(8192, spdlog::async_overflow_policy::block_retry);
```

Comment:
This method indicates that all logging should occur asynchronously. `8192` is the size of the asynchronous queue that is used to process log messages, in elements. This results in a logging buffer that is roughly 800KB in size. For more information, see https://github.com/gabime/spdlog/wiki/6.-Asynchronous-logging

I've added comments to this effect in both locations where this method is called.